packages/types/custom/core.ts

```diff
@@ -592,6 +596,7 @@ export interface MapConfig {
   iconMenu?: IconMenuConfiguration
   pins?: PinsConfiguration
   reverseGeocoder?: ReverseGeocoderConfiguration
+  scale?: ScaleConfiguration
```

Comment:
Please add a changelog entry for this addition including the reference to the newly added interface `ScaleConfiguration`

Reply:
032a181894f6ef38c73fe650f71c0a8c82e77127